dev-vcs/tortoisehg/tortoisehg-6.9.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
```

Comment:
any chance this supports py3_13?

Reply:
The tortoisehg-6.9 "states" it tested with mercurial-6.3 ... 6.8 and last isn't compatible with py3.13. But upstream confirm [1] that tortoisehg-6.9 actually must be compatible with mercurial-6.9 so I need to patch one source file and then retest against py3.13 too.

[1] https://foss.heptapod.net/mercurial/tortoisehg/thg/-/issues/6010#note_378967

Reply:
The ebuild is updated to support mecurial-6.9 + python-3.13: the tests are passed.